src/main/java/by/onliner/constants/OnlinerConstants.java

```diff
@@ -6,4 +6,16 @@
 
     public static final String LOG_PATH = "target/logs/appTest.log";
     public static final String SAVE_VIDEO_PATH = "./Videos/";
+
+    public static final String ABOUT_LINK = "/about";
+
+    public static final String IPHONE = "iphone 13";
```

Comment:
i am guessing it's test data, please separate test data from constants. It's worth storing some test data in the test classes, some test data should be stored in separate files/classes
What do you think how do we choose which approach to use?

Reply:
Moved to the test class, in the future I plan to create test data